src/ralph/cli.py

```diff
@@ -638,21 +631,19 @@ def convert(from_, to_, ignore_errors, fail_on_unknown, **conversion_set_kwargs)
 )
 def read(
     backend,
-    archive,
     chunk_size,
     target,
     query,
     ignore_errors,
     **options,
 ):  # pylint: disable=too-many-arguments
-    """Read an archive or records from a configured backend."""
+    """Read records matching the QUERY (json or string) from a configured backend."""
```

Comment:
It would be the only documentation on the `read` command, which I find a bit light.
Do you know any way to document how to use the `read` methods? We could probably provide examples?

Reply:
That's a great idea! Thanks) Indeed, having dedicated help pages for each backend would be helpful)

I think we could use sub-commands with click to achieve something like:
```
ralph read es --help # getting read docs for es backend
ralph read es --option1 --option2 QUERY
```
Will give it a try in a separate PR)